src/log.ts

```diff
@@ -9,6 +9,8 @@ const template = ({
 	stack,
 	...rest
 }: TransformableInfo): string => {
+	/* eslint-disable-next-line @typescript-eslint/no-magic-numbers --
+	 * "0" here is semantically meaningful*/
```

Comment:
I think if this rule requires us to do this sort of thing, we should leave it off. My line of thinking here is that any time we turn off a rule on a line-by-line level, we create a permission structure for doing that sort of thing, and that will eventually lead to turning off rules out of laziness. Linting should promote consistency and reduce cognitive load, if we start to believe that disabling rules line by line is an option that will cut against those purposes.

Reply:
Yeah I hear you and I'm totally down for that but first just to check: we can disable it for certain numbers (it's common to do it for 0 and 1).  Thoughts on that approach?

I will say, turning on this rule did flag a handful of places where magic numbers that are semantically useful.